build.gradle

```diff
@@ -73,7 +75,7 @@ dependencies {
 
     implementation 'org.jetbrains.kotlin:kotlin-stdlib-jdk8'
 
-    implementation 'io.ktor:ktor-server-netty:1.3.1'
+    implementation 'io.ktor:ktor-server-netty:1.6.8'
```

Comment:
We also need to apply changes from this commit https://github.com/th2-net/th2-rpt-data-provider/commit/90a420221e0d9f1bdd48f8c9aeb0daee82ca7e35